### DISCUSSION TITLE
How to get <note> from Markdown?

### DISCUSSION THREAD

**Discussion by judgeNotFound** [dita-ot/dita-ot#4163] [Q&A] [answered]
We are using Dita by transforming a Markdown document to XML. There, we desire to have note tags in the following format:

```
<note>
    <p>
        Text
    </p>
</note>
```

Sadly, I could not find any information on how to produce such an output in the wiki, but it is present in some documents in the repo. Could you give me a hint, how to reach the output above?

**Comment by jelovirt** [comment, marked as answer]
LwDITA plug-in 5.1 has support for admonition extension, but it's not bundled with any released version of DITA-OT. You can install it from https://www.dita-ot.org/plugins#!org.lwdita by first uninstalling `org.lwdita` plug-in and then reinstalling the latest one.

The admonition extension for your desired output would be

```markdown
!!! note

    Text
```

See https://github.com/jelovirt/org.lwdita/wiki/Markdown-Syntax-reference#notes

**Comment by infotexture** [comment]
@rrcomtech We'll ship this new plug-in version in an upcoming version of DITA-OT, and include information on the new syntax in the documentation when we do. Until then, you can test it as Jarno describes above (and let us know if you notice any issues).

_**P.S.** Nice to see another DITA user in Potsdam…_ 🙂🇩🇪👋
